src/base/reconstruction_test.cc

```diff
@@ -366,12 +362,6 @@ BOOST_AUTO_TEST_CASE(TestCrop) {
   BOOST_CHECK(recon2.IsImageRegistered(1));
   BOOST_CHECK(recon2.IsImageRegistered(2));
   BOOST_CHECK(!recon2.IsImageRegistered(3));
-  BOOST_CHECK_LE(
```

Comment:
It was not a stable check since the points are in an unordered map and there is no guarantee that they will be included in the same order as they appeared in the original reconstruction. The test was passing on Windows and failing on all other platforms due to this non-deterministic behavior.

I decided to remove this test instead of writing more tedious code along the lines of ensuring that at least one of the included points matches the target and also it's not double counted.

Reply:
Okay, thanks.